synapse/rest/client/v2_alpha/_base.py

```diff
@@ -26,7 +26,7 @@
 logger = logging.getLogger(__name__)
 
 
-def client_patterns(path_regex, releases=(0,), unstable=True, v1=False):
+def client_patterns(*path_regexes, releases=(0,), unstable=True, v1=False):
```

Comment:
This originally was going to be added functionality to assist this PR, but the need was removed.

This seems like a nice thing to have though (be able to specify multiple regexes to `client_patterns`) so we may want to leave it in anyways?

Reply:
I'm not keen on registering the same servlet at multiple paths anyway, so I'd rather it went away rather than encouraging such naughtiness.

Reply:
It came in handy for the submitToken endpoint for which the same code could be used for both email and msisdn. We could have two servlets that run the same method, though that would mean putting the method in another class/not in a class.

Reply:
We need to add another endpoint for just adding an email to your account. We should probably split this function up into multiple servlets.